src/modules/routes/components/routes/routes.jsx

```diff
@@ -20,6 +20,7 @@ const Routes = p => (
     <Route path={makePath(VIEWS.MARKET)} component={COMPONENTS.Market} />
     <Route path={makePath(VIEWS.CONNECT)} component={COMPONENTS.Connect} />
     <Route path={makePath(VIEWS.CREATE)} component={COMPONENTS.Create} />
+    <Route path={makePath(VIEWS.REPORT)} component={COMPONENTS.Report} />
```

Comment:
I'm thinking this should be an AuthenticatedRoute. And for that matter, unrelated to this PR, the route for `CREATE` (one line above) should be too. @stephensprinkle is that right?

Reply:
For `REPORT`, yes, it should be authenticated.  

`CREATE` is actually for account creation…which ultimately is going away, but for now it should be a regular route.